src/index.html

```diff
@@ -1,13 +1,424 @@
+              <div class="contacts__info--place">
+                <p>
+                  The Met Fifth Avenue
+                </p>
+              </div>
+              <div class="contacts__info-element">
+                <div class="contacts__info--title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 212-535-7710"
+                  class="contacts__info--link"
+                >
+                  +1 212-535-7710
+                </a>
+              </div>
+              <div class="contacts__info--element">
+                <div class="contacts__info--title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/M1zJH8Q4cCdQLr7S6"
+                  class="contacts__info--link"
+                  target="_blank"
+                >
+                  1000 Fifth Avenue <br>
+                  New York, NY 10028
+                </a>
+              </div>
+            </div>
+            <div class="contacts__info">
+              <div class="contacts__info--place">
+                <p>
+                  The Met Cloisters
+                </p>
+              </div>
+              <div class="contacts__info-element">
+                <div class="contacts__info--title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 212-923-3700"
+                  class="contacts__info--link"
+                >
+                +1 212-923-3700
+                </a>
+              </div>
+              <div class="contacts__info--element">
+                <div class="contacts__info--title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/zBvgJwEiqgNnDmDN6"
+                  class="contacts__info--link"
+                  target="_blank"
+                >
+                  99 Margaret Corbin Drive <br>
+                  Fort Tryon Park <br>
+                  New York, NY 10040
+                </a>
+              </div>
+            </div>
+            <form
+              action="#"
+              method="get"
+              class="contacts__form form"
+            >
+              <input
+                name="name"
+                type="text"
+                placeholder="Name"
+                class="contacts__field form-input"
+                required
+              >
+              <input
+                name="email"
+                type="email"
+                placeholder="Email"
+                class="contacts__field form-input"
+                required
+              >
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="
+                contacts__field
+                form-input
+                form-input--textarea"
+                required
+              ></textarea>
+              <button class="button">
```

Comment:
your button do nothing on cklick

Reply:
maybe you mean this, because button reload page on click
![Screenshot_1](https://user-images.githubusercontent.com/108741138/181894231-f65ec43d-24f6-4e64-b1dd-328eca26365d.png)